docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -11,14 +11,16 @@
     "(realtime-monitor-drift-tutor)=\n",
```

Comment:
<p>remove the base period, or explain that the default value is <code>10m</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
deleted